source/security/encryption/minio-kes.rst

```diff
@@ -18,67 +18,338 @@ key-management system for high-performance applications. KES provides
 a bridge between applications running in bare-metal or orchestrated
 environments to centralised KMS solutions. 
 
-<DIAGRAM>
+.. image:: https://raw.githubusercontent.com/minio/kes/master/.github/arch.png
+   :alt: The KES server sits between the MinIO deployment and the KMS.
+   :width: 90%
+   :align: center
 
 KES is designed for simplicity, scalability, and security. It requires 
 minimal configuration to enable full functionality and requires only
 basic familiarity with cryptography or key-management concepts.
 
 MinIO servers require KES for performing Server-Side Encryption (SSE) of objects
-using Key Management Services (KMS). 
+using Key Management Services (KMS). Applications *may* use KES to support
+cryptographic key operations independent of MinIO. 
 
-KES Server Process
-------------------
+This page documents conceptual information about MinIO KES. For documentation
+on the :mc:`kes` command line tool, see the
+:doc:`KES reference page </minio-cli/minio-kes-reference>`.
 
-.. mc:: kes server
+.. _minio-kes-getting-started:
 
-:mc:`kes server` command starts the KES server. The :mc:`kes server` process
-handles requests for creating and retrieving cryptography keys from a supported
-Key Management System (KMS). 
+Getting Started
```

Comment:
@aead this is also rough but could use an eyeball. Getting Started + IAM-related information on KES. Server API section is just for quick reference. Will likely import the Server API page - though maybe its better in the wiki? hard to say.